ui/src/criteria/classification.tsx

```diff
@@ -132,6 +132,37 @@ function keyForNode(node: ClassificationNode): DataKey {
   return key;
 }
 
+type SearchData = {
```

Comment:
- `SearchData.query` stores the text string that the user typed into the search box on the list view.
- `SearchData.hierarchy` stores the id that is selected in the hierarchy view, which is the row the user clicked on in the list view to get there.

Am I following this correctly? And if so, could you please add a comments about this for future reference.

Reply:
Done.